app/cafe.py

```diff
@@ -13,9 +13,10 @@ def visit_cafe(self, visitor: dict) -> str:
             raise NotVaccinatedError("All friends should be vaccinated")
         if visitor["vaccine"]["expiration_date"] < datetime.date.today():
             raise OutdatedVaccineError("OutdatedVaccine")
-        if visitor["wearing_a_mask"] is False:
-            raise NotWearingMaskError(
-                f"Friends should buy {masks_to_buy} masks"
-            )
+        if "wearing_a_mask" in visitor:

```

Comment:
use `get` method https://www.w3schools.com/python/ref_dictionary_get.asp instead of this code